apps/web/app/error.tsx

```
@@ -5,11 +5,32 @@ import { Button } from "@/modules/ui/components/button";
+/**
+ * Get translated error messages based on error type
+ * All translation keys are directly visible to Tolgee's static analysis
+ */
+const getErrorMessages = (type: ClientErrorType, t: (key: string) => string) => {
+  switch (type) {
+    case "rate_limit":
+      return {
+        title: t("common.error_rate_limit_title"),
+        description: t("common.error_rate_limit_description"),
+      };
+    case "general":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)